e2e/tests/popupClickHandling-example.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Button, Popup } from '@stardust-ui/react'
+
+export const selectors = {
+  triggerButtonId: 'trigger',
+  popupContentClass: Popup.slotClassNames.content,
+  popupContentButtonId: 'content-button',
+}
+
+const PopupClickHandlingExample = () => {
+  const [show, setShow] = React.useState(true)
+  return (
+    <>
```

Comment:
Sorry, I was not clear there: we don't need a fragment there at all because there is a single child

Reply:
got your point, thanks - yes, was just copied from the issue's example. Thanks!